incuna-sass/mixins/_triangle.sass

```diff
@@ -0,0 +1,96 @@
+// CSS TRIANGLE
+// Include this mixin on the element you want to have a triangle on it
+// By default the arrow is centered on the bottom of an element, change mixin
+// options to amend it's location and position
+
+// Mixin arguments
+// ==================================
+// $color: hex/variable/named color.
+//    -- default: #000
+// $width: px/rem(px)
+// $height: px/rem(px)
+//    -- default: 10px
+//    Increase width to create a wider arrow or height to 
+//    create a narrower arrow. Include the unit in the argument.
+// $location: bottom | top | left | right
+//    -- default: bottom
+// $position: centered | from-top | from-bottom | from-left | from-right
+//    -- default: centered
+// $position-value: px/rem(px)
+//    -- default: none
+//    If using the default of centered, this is not required
+//    If using any other position, this is required in px/rem(px)/%
+// $pseudo: before | after
+//   -- default: after
+
+// Mixin use examples
+// ==================================
+// 15px triangle on the bottom middle of an element:
+// @include triangle(green, 15px, 15px)
+//
+// 10px triangle on the right middle of an element:
+// @include triangle (green, 10px, 10px, right)
+//
+// 15px triangle on the left of an element, 20px from the top:
+// @include triangle(green, 15px, 15px, left, from-top, 20px)
+//
+// Narrow tall triangle on the bottom of an element, 5px from the right:
+// @include triangle(green, 5px, 15px, bottom, from-right, 5px)
+
+@mixin triangle($color: #000, $width: 10px, $height: $width, $location: bottom, $position: centered, $position-value: none, $pseudo: after)
+    position: relative
+    &::#{$pseudo}
+        content: ''
+        height: 0
+        width: 0
+        position: absolute
+
+        @if ($location == "bottom" or $location == "top")
+            @if $position == "centered"
+                @include transform(translateX(-50%))
+                left: 50%
+            @else if $position == "from-left"
+                left: $position-value
+            @else if $position == "from-right"
+                right: $position-value
+
+        @if $location == "bottom"
+            top: 100%
+            border: solid transparent
+                width: $width
+                top-width: $height
+                bottom-width: 0
+                top-color: $color
+
+        @if $location == "top"
+            bottom: 100%
+            border: solid transparent
+                width: $width
+                bottom-width: $height
+                top-width: 0
```

Comment:
the ordering of the top / bottom values is different here to the one above, it's not majorly important, but since it's quite a complex mixin maybe it would be better to have consistent order